cryptostock/account/tests/test_create_salesdashboard.py

```diff
@@ -0,0 +1,67 @@
+import decimal
+
+from account.tests.conftest import AssetFactory, WalletRecordFactory
+
+
+def test_valid_create_sales_dashboard(auth_broker, broker_account):
+    wallet_record = WalletRecordFactory(wallet=broker_account.wallet)
+    asset = wallet_record.asset
+    asset_count_before_create = broker_account.wallet.wallet_record.get(
+        asset=asset
+    ).count
+    data = {"asset": asset.id, "count": 100.4444, "price": 500.555555}
+    response = auth_broker.post("/api/salesdashboard/", data=data)
+    asset_count_after_create = broker_account.wallet.wallet_record.get(
+        asset=asset
+    ).count
+    assert response.status_code == 201
+    assert asset_count_after_create == asset_count_before_create
+    assert response.json() == {
+        "id": 1,
+        "asset": {"id": asset.id, "name": asset.name, "description": asset.description},
+        "count": str(data["count"]),
+        "price": str(data["price"]),
+        "broker": {
+            "id": broker_account.id,
+            "name": broker_account.name,
+            "owner": broker_account.owner.username,
+            "wallet": {
+                "id": broker_account.wallet.id,
+                "name": broker_account.wallet.name,
+            },
+        },
+    }
+
+
+def test_create_sales_dashboard_not_broker(auth_client):
+    asset = AssetFactory()
+    data = {"asset": asset.id, "count": 100.4444, "price": 500.555555}
+    response = auth_client.post("/api/salesdashboard/", data=data)
+    assert response.status_code == 400
+    assert response.json() == [
+        "You are not a broker. You haven`t permissions for this operation."
+    ]
```

Comment:
Split test phases(arrange, act, assert) with newlines:
```suggestion
    asset = AssetFactory()

    data = {"asset": asset.id, "count": 100.4444, "price": 500.555555}
    response = auth_client.post("/api/salesdashboard/", data=data)
    
    assert response.status_code == 400
    assert response.json() == [
        "You are not a broker. You haven`t permissions for this operation."
    ]
```

Reply:
done